README.md

```diff
@@ -1,113 +1,100 @@
 [![npm version](https://badge.fury.io/js/@bucketeer%2Fnode-server-sdk.svg)](https://badge.fury.io/js/@bucketeer%2Fnode-server-sdk)
 
-# Bucketeer Server-Side SDK for Node.js
+# Bucketeer Server-Side SDK for Node JS
 
-## Table of Contents
+[Bucketeer](https://bucketeer.io) is an open-source platform created by [CyberAgent](https://www.cyberagent.co.jp/en) to help teams make better decisions, reduce deployment lead time and release risk through feature flags. Bucketeer offers advanced features like dark launches and staged rollouts that perform limited releases based on user attributes, devices, and other segments.
 
- * [Installation](#installation)
- * [Example](#example)
- * [Documentation](#documentation)
- * [Contributing](#contributing)
- * [Development](#development)
+[Getting started](https://docs.bucketeer.io/getting-started) using Bucketeer SDK.
 
 ## Installation
 
-The Bucketeer server-side SDK for Node.js is available on npm as `@bucketeer/node-server-sdk`:
+See our [documentation](https://docs.bucketeer.io/sdk/server-side/node-js) to install the SDK.
 
-```bash
-$ npm install --save @bucketeer/node-server-sdk
-```
+## Contributing
+
+We would ❤️ for you to contribute to Bucketeer and help improve it! Anyone can use and enjoy it!
+
+Please follow our contribution guide [here](https://docs.bucketeer.io/contribution-guide/contributing).
 
 ## Example
 
-Before building an example, you need some setups.
+Before building the example, please follow the following steps.
 
-- Replace placeholders in [index.ts](./example/src/index.ts)
-- Move to example directory. `cd example`
-- Install dependencies. `make init`
+- Ensure that you have the `yarn` installed.
+- Configure the `host` and the `token` info in the [index.ts](https://github.com/bucketeer-io/node-server-sdk/blob/master/example/src/index.ts#L15-L19).
+- Move to the example directory `cd example`.
+- Install dependencies `make init`.
 
-Build and start example server.
+### Build and start an example server
 
 ```bash
 make start
 ```
 
-If you want to use published SDK instead of local one, see `NOTE:` in [the example code](./example/src/index.ts)
-
-## Documentation
-
-- [Tutorial](https://bucketeer.io/docs/#/./server-side-sdk-tutorial-node)
-- [Integration](https://bucketeer.io/docs/#/./server-side-sdk-reference-guides-node)
-
-## Contributing
-
-[CONTRIBUTING.md](./CONTRIBUTING.md)
+If you want to use a published SDK instead of a local one, replace the line where it imports the library in the [example code](https://github.com/bucketeer-io/node-server-sdk/blob/master/example/src/index.ts#L7-L8)
 
 ## Development
 
 ### Setup
 
-Install prerequisite tools.
+Install the prerequisite tools.
 
 - [nvm](https://github.com/nvm-sh/nvm)
 - [Yarn](https://yarnpkg.com/en/docs/install)
 
-Setup node version.
+#### Setup node version
 
 ```bash
-nvm use
+nvm use 16.11.12
```

Comment:
`nvm use` is enough because it refers to [here](https://github.com/bucketeer-io/node-server-sdk/blob/5ab6c117f608dfdd6e54735bda8a87a5b4358bf5/.nvmrc#L1)